content/integrations/azure.md

```diff
@@ -103,62 +101,73 @@ azure role assignment create --objectId {object-Id} --roleName Reader --subscrip
-10. Click on "Configure" (Make sure you are on the page for the app, not the active directory) <img src="/static/images/azure/configure_click.png" style="width: 50%;display:block;"/>
-11. Scroll down to Client ID. Copy and paste the value into the form on the Azure setup tile <img src="/static/images/azure/client_id_form.png" style="width: 50%;display:block;display:block;"/>
-12. Scroll down to Keys and create a new key (1 or 2 years are both acceptable)
-	- Note: the client secret key will be shown after you save
-13. Scroll down to "Permissions To Other Applications"
-14. Click Add Application
-15. Select Windows Azure Service Management API and click the check mark on the bottom right of the screen
-16. Under Delegated Permissions, check off “Access Azure Service Management as organization users (preview)” <img src="/static/images/azure/app_permissions_dropdown.png" style="width: 50%;display:block;"/>
-17. Click "Save" at the bottom of the screen 
-18. Scroll back up to Keys, and copy and paste the generated Client Secret key into the form on the Azure setup tile <img src="/static/images/azure/client_secret_form.png" style="width: 50%;display:block;display:block;"/>
-19. Click "Install Integration" to complete the application creation process
+1. Navigate to the "App Registrations" tab within your Azure Active Directory.
+2. Press "Add"
+3. Enter a name and Sign-on URL for this app. 
+  * These will NOT be used in any way and are simply required as part of the setup process. 
+  * Leave Application "Type as Web app/ API"
+4. Press "Create"
+
+    ![settings](/static/images/azure/Azure_create_ad.png)
+
+5. Once it is created, select the App from the list of App Registrations
+6. Copy the "Application ID" and paste the value into "Client ID" in the form on the Azure setup tile
+    
+    ![settings](/static/images/azure/Azure_client_id.png)
+
+    ![settings](/static/images/azure/client_id_form.png)
+
+7. For the same app, go to "All settings"
+8. Go to "Keys"
+9. Enter a new Client Secret key and press Save
+  * Make sure to note when the key will expire!
+10. When the Secret Key is shown, copy and paste it in "Client Secret" in the form on the Azure setup tile
+
+    ![settings](/static/images/azure/Azure_client_secret.png)
+
+    ![settings](/static/images/azure/client_secret_form.png)
+
+11. Click "Install Integration" to complete the application creation process
 
 ##### Giving Read Permissions to the Application
 {: #installation3}
 
-1. Navigate to [portal.azure.com][2]
-2. Navigate to “Subscriptions” on the left hand menu
+1. Navigate to “Subscriptions” on the left hand menu
 
     ![settings](/static/images/azure/subscriptions_icon.png){:style="width:50%;"}
 
-3. Click on the subscription you would like to monitor
-4. Click on "Access control (IAM)" in the lefthand menu 
+2. Click on the subscription you would like to monitor
+3. Click on "Access control (IAM)" in the lefthand menu 
 
```

Comment:
👍  think it was from a larger chunk of code I deleted and reordered. Looks good on Rake